frog/hardware/plugins/stepper_motor/st10_controller.py

```diff
@@ -281,6 +283,18 @@ def _check_device_id(self) -> None:
         if self._read_sync() != self.ST10_MODEL_ID:
             raise ST10ControllerError("Device ID indicates this is not an ST10")
 
+    def _disable_limit_switches(self) -> None:
+        """Disable the limit switches on the controller.
+
+        Without this, we get an error if the motor hits a limit switch. The limit
+        switches aren't actually needed to protect anything in either the FINESSE or
+        UNIRAS rigs (the worst that will happen is the motor will just spin around), so
+        we don't need them.
+        """
+        # "Define limit". This command allows for setting the behaviour of the limit
+        # switches. The "3" option means that they are treated as normal inputs.
```

Comment:
> The "3" option means that they are treated as normal inputs.

I can't quite work out what this means. Why does this effectively disable the limits?

Reply:
Ah, OK, fair enough!